src/convertToObject.js

```diff
@@ -6,7 +6,11 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resut = {};
+
+  const array = sourceString.split(';');
```

Comment:
The variable name 'array' is not descriptive and should be replaced with a name that reflects its purpose, such as 'styleDeclarations' or 'cssDeclarations'.